Index: modules/tome_static/src/Commands/StaticCommand.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/modules/tome_static/src/Commands/StaticCommand.php b/modules/tome_static/src/Commands/StaticCommand.php
--- a/modules/tome_static/src/Commands/StaticCommand.php
+++ b/modules/tome_static/src/Commands/StaticCommand.php	(date 1732036491000)
@@ -175,9 +175,11 @@
     $show_progress && $this->io->progressStart(count($paths));

     $invoke_paths = [];
+    \Drupal::logger('tome-collect-count')->info(count($commands) . ' commands given');
     $collected_errors = $this->runCommands($commands, $process_count, $retry_count, function (Process $process) use ($show_progress, &$invoke_paths, $path_count) {
       $show_progress && $this->io->progressAdvance($path_count);
       $output = $process->getOutput();
+      \Drupal::logger('tome-collect-process')->info(':output:' . strlen($output) . ':commandline:' . $process->getCommandLine() . ':output:' . $output);
       if (!empty($output) && $json = json_decode($output, TRUE)) {
         $invoke_paths = array_merge($invoke_paths, $json);
       }
